scripts/ci/build-installers.sh

```diff
@@ -44,6 +44,10 @@ if [ -z "$ARGV_OPERATING_SYSTEM" ] || [ -z "$ARGV_ARCHITECTURE" ]; then
   usage
 fi
 
+if [ "${GH_TOKEN:-false}" != "false" ]; then
```

Comment:
The issue is that secure environment variables, like `GH_TOKEN`, don't get applied during pull request builds, while non secure variable do (like `PUBLISH`). Setting `GH_TOKEN` and `PUBLISH` in the Etcher project meant that during pull requests, `PUBLISH` would be present while `GH_TOKEN` would be missing, causing `electron-builder` to complain about not being able to publish because there was no token.

I guess we can fix this by checking the opposite case: If `PUBLISH` is enabled, but there is no `GH_TOKEN`, we show a warning and unset `PUBLISH`